docs/consume-packages/Package-Source-Mapping.md

```diff
@@ -111,3 +111,4 @@ For an idea of how your source mappings may look like, refer to our [samples rep
 >
 > * There are no nuget.exe or dotnet.exe commands for managing the package source mapping configuration, see [NuGet/Home#10735](https://github.com/NuGet/Home/issues/10735).
 > * There are no means of mapping packages at package installation time, see [NuGet/Home#10730](https://github.com/NuGet/Home/issues/10730).
+> * There is a limitation when using the `DotNetCoreCLI@2` Azure Pipelines task which can be worked around by using `feed-` prefixes in your source mapping configuration, it is recommended however to use `NuGetAuthenticate` for your authentication needs instead, see [microsoft/azure-pipelines-tasks#15542](https://github.com/microsoft/azure-pipelines-tasks/issues/15542).

```

Comment:
```suggestion
> * There is a limitation when using the `DotNetCoreCLI@2` Azure Pipelines task which can be worked around by using `feed-` prefixes in your source mapping configuration. It is recommended however to use `NuGetAuthenticate` for your authentication needs and call the dotnet cli directly from a script task. See [microsoft/azure-pipelines-tasks#15542](https://github.com/microsoft/azure-pipelines-tasks/issues/15542).
```

This feels like a run-on-sentence, so I suggest breaking it into two (or more) sentences.

The second part of my suggestion is that people inexperienced with Azure DevOps might have the impression that `NuGetAuthenticate` is used in the same scenarios as `DotNetCoreCli`, which is not the case. Therefore, maybe being explicit about needing to run the dotnet cli from a script task helps be clearer what, exactly, the recommendation is.